src/klein/_dihttp.py

```diff
@@ -49,7 +47,7 @@ def urlFromRequest(request):
     )
 
 
-@provider(IRequiredParameter, IDependencyInjector)
+@provider(IRequiredParameter, IDependencyInjector)  # type: ignore[misc]
```

Comment:
When we do the ZI mypy plugin thing, will it obviate (and start warning on) all these `ignore[misc]`s? I want to make sure we can find these again easily to clean them up, and normally I’d say “drop a comment in there” but if we don’t need a comment then great.

Reply:
I don't think we need a comment; unused ignored generate a warning.

In any case, I'm going to want to eventually search for all of the ignores and see about cleaning them up or filing `mypy` bugs.